cypress/e2e/search.cy.ts

```diff
@@ -0,0 +1,40 @@
+import { NAVIGATION } from "../elements/navigation";
+import { SEARCH_PAGE_SELECTORS } from "../elements/search-page";
+
+describe("Search for products", () => {
+  beforeEach(() => {
+    cy.visit("/");
+  });
+  it("should search for products SRS_0405", () => {
+    const typedText = "polo";
```

Comment:
Do we want to hardcode text here? Maybe there is an option to take the name from some variable?

Reply:
I added fixtures